server/master/src/main/java/org/apache/accumulo/master/replication/RemoveCompleteReplicationRecords.java

```diff
@@ -57,9 +58,11 @@
   private static final Logger log = LoggerFactory.getLogger(RemoveCompleteReplicationRecords.class);
 
   private Connector conn;
+  private Master master;
 
-  public RemoveCompleteReplicationRecords(Connector conn) {
+  public RemoveCompleteReplicationRecords(Connector conn, Master master) {
```

Comment:
bq. Is this connector for the remote peer?

No, it's for the "local" instance. This is just cleaning up stuff in the accumulo.replication table on the "source" instance.